kyaari-oms/src/dashboards/vendors/pages/Performance.tsx

```
@@ -103,57 +107,179 @@ const orderFulfillmentData = [
-          <div className={`flex items-center gap-1 text-sm font-medium ${
-            trend === 'up' ? 'text-green-600' : 'text-red-600'
-          }`}>
-            {trend === 'up' ? <TrendingUpIcon size={16} /> : <TrendingDownIcon size={16} />}
-            {Math.abs(change)}%
+  const toCSV = (rows: Array<Record<string, unknown>>, headerOrder?: string[]) => {
+    if (!rows || rows.length === 0) return ''
+    const headers = headerOrder && headerOrder.length > 0
+      ? headerOrder
+      : Array.from(new Set(rows.flatMap((r) => Object.keys(r))))
+
+    const escape = (value: unknown) => {
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)